utils/strings.js

```diff
@@ -2,6 +2,10 @@ const _ = require('lodash');
 
 module.exports = {
   pascalCase(input) {
+    if (/^[\d]/g.exec(input)) {
+      return `Table${input}`;
+    }
```

Comment:
[Consistency] Not convinced this is a good idea to place such specific code logic in a function named "pascalCase" and that now does not exactly "pascalCase" the input value.